brax/envs/__init__.py

```diff
@@ -57,3 +57,16 @@ def create_fn(env_name: str, **kwargs) -> Callable[..., Env]:
 def create_gym_env(env_name: str, **kwargs) -> gym.Env:
   """Creates a Gym Env with a specified brax system."""
   return wrappers.GymWrapper(create(env_name, **kwargs))
+
+
+def create_gym_vector_env(env_name: str, **kwargs) -> gym.vector.VectorEnv:
```

Comment:
Rather than create a new function, I think we can just check the batch_size parameter in `create_gym_env` and return `VecGymWrapper` if it's > 0

Reply:
Might I suggest this:
```python3
from typing import overload

@overload
def create_env(env_name: str, **kwargs) -> GymWrapper:
    ...

@overload
def create_env(env_name: str, batch_size: int, **kwargs) -> VecGymWrapper:
    ...

def create_env(
    env_name: str, batch_size: int = None, **kwargs
) -> Union[GymWrapper, VecGymWrapper]:
    if batch_size is None:
        return create_gym_env(env_name=env_name, **kwargs)
    else:
        return create_gym_vector_env(env_name=env_name, batch_size=batch_size, **kwargs)
```